docs/howto/openstack/nova/move-server-between-regions.md

```diff
@@ -0,0 +1,221 @@
+| snapshot_id         | None                                 |
+| source_volid        | 14942b23-....-....-....-............ |
+| status              | creating                             |
+| type                | default                              |
+| updated_at          | None                                 |
+| user_id             | ................................     |
++---------------------+--------------------------------------+
+```
+
+## Creating an image of a volume
+
+Then create the image, by using the following command:
+
+```bash
+openstack image create --volume <volume_id> <new_image_name>
+```
+
+> Substitute `<volume_id>` with the ID from the newly created volume in the previous step.
+
+After a while you will get a printout showing you information of the new image, such as the current status and the image ID. Depending on the size of the volume it might take a while to upload and while it is the image `status` will be `uploading`
+
+```plain
++---------------------+--------------------------------------+
+| Field               | Value                                |
++---------------------+--------------------------------------+
+| container_format    | bare                                 |
+| disk_format         | raw                                  |
+| display_description | None                                 |
+| id                  | 9e171aa3-....-....-....-............ |
+| image_id            | de7e1fcc-....-....-....-............ |
+| image_name          | <new_image_name>                     |
+| protected           | False                                |
+| size                | 20                                   |
+| status              | uploading                            |
+| updated_at          | 2022-11-10T15:22:12.000000           |
+| visibility          | shared                               |
+| volume_type         | default                              |
++---------------------+--------------------------------------+
+```
+
+Before you continue, make sure the image `status` field is `active`, otherwise wait a bit and then check again with:
+
+```bash
+openstack image show -c status <image_id>
+```
+
+The printout should look like this before you continue.
+
+```plain
++--------+--------+
+| Field  | Value  |
++--------+--------+
+| status | active |
++--------+--------+
+```
+
+> It is now safe to remove the volume you created earlier.
+
+## Downloading an image
+
+Download the image to the local machine, using the following command:
+
+```bash
+openstack image save --file <local_image_name>.<disk_format> <image_id>
+```
+
+> Substitute `<disk_format>` with the value from the printout in the previous step.  
+> Substitute `<image_id>` with the ID from the printout in the previous step.
+
+When you have downloaded the file, you are done with the steps for the source region. The following steps will be done on the target region.
+
+> It is now safe to remove the image you created earlier.
```

Comment:
Yeah, maybe it's not. :) You don't know whether the image you downloaded is broken or not. Maybe try `md5sum` against the downloaded file, and compare it to the image's `checksum` value?

Reply:
I cannot find any command to have openstack display the checksum of an image.

`openstack image show -c checksum <image_id>`
```
No recognized column names in ['checksum']. Recognized columns are ('container_format', 'created_at', 
'disk_format', 'file', 'id', 'min_disk', 'min_ram', 'name', 'owner', 'properties', 'protected', 'schema', 
'size', 'status', 'tags', 'updated_at', 'visibility').
```